src/translations/en.json

```diff
@@ -362,5 +362,7 @@
     "insertNetworkModificationNodeBefore": "Insert a network modification node before",
     "insertModelNodeBefore": "Insert a model node before",
     "insertNetworkModificationNodeAfter": "Insert a network modification node after",
-    "insertModelNodeAfter": "Insert a model node after"
+    "insertModelNodeAfter": "Insert a model node after",
+
+    "LoadStudyAndParentsInfoError": "An error occured while loading current study's metadata"
```

Comment:
metadata is something else, avoid confusion I think.

Reply:
fixed